### PR TITLE
Stackgenie details added

### PR DESCRIPTION
**Pull request by ebinbabu**
Stackgenie details adding to istio ecosystem

### CODE REVIEW

**Comment by kfaseela** [comment]
@ebinbabu could you please sign the CLA?

File: data/companies.yml [hunk 1]
```diff
@@ -330,6 +330,11 @@ pro_services:
     details:
       - "Add information about your how your company supports Istio."
       - "You can add multiple paragraphs."
+  - name: "Stackgenie"
+    logo: "/logos/stackgenie.svg"
+    url: "https:stackgenie.io"
```
Comment (ericvn) [inline data/companies.yml:335]:
```suggestion
    url: "https://stackgenie.io"
```

File: data/companies.yml [hunk 2]
```diff
@@ -330,6 +330,11 @@ pro_services:
     details:
       - "Add information about your how your company supports Istio."
       - "You can add multiple paragraphs."
+  - name: "Stackgenie"
```
Comment (ericvn) [inline data/companies.yml:333]: Move above the "Your Name Here" entry.

File: data/companies.yml [hunk 3]
```diff
@@ -330,6 +330,11 @@ pro_services:
...
+    logo: "/logos/stackgenie.svg"
+    url: "https:stackgenie.io"
+    description: "StackGenie specialises in assisting companies with Istio adoption. Istio consulting and advising, Istio production installation, configuration and integration with observability tools, security measures, and enterprise Istio support and training are all part of our entire professional services offering. Leveraging our extensive expertise in cloud infrastructures and Kubernetes projects, we provide end-to-end services to our clients.
+To learn more, please visit www.stackgenie.io."
```
Comment (kfaseela) [inline data/companies.yml:337]:
```suggestion
  - name: "Stackgenie"
    logo: "/logos/stackgenie.svg"
    url: "https://www.stackgenie.io"
    description: "StackGenie specialises in assisting companies with Istio adoption. Istio consulting and advising, Istio production installation, configuration and integration with observability tools, security measures, and enterprise Istio support and training are all part of our entire professional services offering.
    details:
        - "Leveraging our extensive expertise in cloud infrastructures and Kubernetes projects, we provide end-to-end services to our clients."
```

Comment (kfaseela) [reply data/companies.yml:337]: This whole section should move above L326

File: data/companies.yml [hunk 4]
```diff
@@ -323,14 +323,20 @@ pro_services:
...
+      - "Istio consulting and advising, Istio production installation, configuration and integration with observability tools, security measures, and enterprise Istio support and training are all part of our entire professional services offering."
+      - "Leveraging our extensive expertise in cloud Native and Kubernetes projects, we provide end-to-end services to our clients."
   - name: "Your Name Here"
     logo: "/logos/istio.svg"
     url: "https://istio.io/"
     description: "Submit a PR to add your company here!"
     details:
       - "Add information about your how your company supports Istio."
       - "You can add multiple paragraphs."
-
```
Comment (ericvn) [inline data/companies.yml:333]: Would you add the separator line back in. Thanks.

**Comment by ericvn** [comment]
Add the line after the "your Name Here" section, to separate there.

File: data/companies.yml [hunk 5]
```diff
@@ -323,14 +323,21 @@ pro_services:
...
```
Comment (kfaseela) [inline data/companies.yml:333]: can you add this newline back?

Comment (ericvn) [reply data/companies.yml:333]: Line 340 is the line, but it has trailing spaces.

Comment (ebinbabu) [reply data/companies.yml]: done ,please review

**Review by kfaseela — APPROVED** [review]

**Comment by kfaseela** [comment]
Thanks!

**Pull request merged**